Sources/Internal/CodableTimestamp.swift

```diff
@@ -52,10 +52,9 @@ extension CodableTimestamp {
       .firstMatch(in: timestampString, range: NSRange(location: 0,
                                                       length: timestampString.count)) !=
       nil else {
-      FirebaseLogger.dataConnect
-        .error(
-          "Timestamp string format doesn't support."
-        )
+      DataConnectLogger.error(
+        "Timestamp string \(timestampString, privacy: .private) format doesn't support."
```

Comment:
I don't think this needs to be private. Its only printing a timestamp - won't contain identifiable info.